src/db.py

```diff
@@ -0,0 +1,57 @@
+import pymysql
+import time
+
+
+def open_connection():
+    connection = pymysql.connect(
+        host='68.225.98.149',
+        user='HQ',
+        passwd='Un4lappable',
```

Comment:
Don't commit passwords. It's possible to remove these from git now, but it's assumed as soon as you've put a password on git that it's compromised. Gonna need to change this one